R/tm_g_ae_oview.R

```diff
@@ -16,67 +16,40 @@
 #' @export
 #'
 #' @examples
-#' library(nestcolor)
+#' data <- teal.data::cdisc_data() |>
```

Comment:
Don't need to prefix `cdisc_data()`
```suggestion
#' data <- cdisc_data() |>
```

Reply:
So no `teal::` and `teal.transofrm` either?

Reply:
Everything (teal.slice, teal.data, teal.transform) is given to the app developer by default when loading teal, it is in most of the cases concise in the `teal.module.xyz` repos, that nothing is called with namespace prefix (unless necessary). They (app developers) tend to understand `library` calls more. It has been something we had debates many times and each time had a consensus that:
- no `::` in the examples
- only `::` in the code (except `base` and `$this` package and few exceptions like `shiny`)
- **`teal::init` is an institution** and can be prefixed ;]

Reply:
All right. This is the first time I hear about this consensus (13 months on the project).
I thought it makes sense to prefix foreign functions in examples so that the reader knows where the function comes from.

The examples (before my meddling) had `osprey::rADSL`, etc. How does that fit into this?

Reply:
This is a grey zone, feel free to remove `osprey::` if it bothers you, I don't have anything against.

Reply:
All or nothing.

Reply:
Just passing my opinion about datasets

> I know rADSL comes from osprey that is loaded as this packaged Depends on it. However we put osprey:: and other package names in front of datasets, so that there is no conflict in the namespace. rADSL and similar like rADLB exist in a couple other packages, so prefixing a package in here makes it clear what is the source of the data.

Agree to remove `::` in front of all other functions. Just wanted to highlight we can have multiple packages loaded with the same dataname called `rADSL`. For now they are the same. I dont think they ever will be different. Just wanted to say that if user hits `?rADSL` and see a couple of packages, the prefix in the example would point him to which dataset we wanted to use. But I agree that this is a grey zone and I don't argue much to stick to my view.